### PR TITLE
fix: update timeout for OCSP call to 2 seconds

### PR DESCRIPTION
**Pull request by priteshbandi**
updated timeout for OCSP call to 2 seconds.

Updates based on updates in https://github.com/notaryproject/notaryproject/pull/249

### CODE REVIEW

**Review by gokarnm — APPROVED** [review]
LGTM!

**Review by shizhMSFT — APPROVED** [review]
LGTM. Should we merge this PR after merging https://github.com/notaryproject/notaryproject/pull/249?

**Comment by yizha1** [comment]
Agree with @shizhMSFT , we need to merge the [spec](https://github.com/notaryproject/notaryproject/pull/249) first, then the implementation changes.

**Pull request merged**
